python_common_logger/src/context/context_handler.py

```diff
@@ -1,28 +1,24 @@
+import contextvars
 from .execution_context import ExecutionContext
 
-from threading import local
+# Define a context variable for execution context
+execution_context_var = contextvars.ContextVar('execution_context', default=ExecutionContext({}))
```

Comment:
Minor, no need to add suffix: _var

Reply:
we already have an arg execution_context here 
`def update_execution_context(execution_context: ExecutionContext, reset=False) -> ExecutionContext:`
because of it using execution_context_var